File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/create-new-context-request/cpp/sync.cpp
absmartly::ContextConfig config;
config.units = {
    {"session_id", "bf06d8cb5d8137290c4abb64155584fbdb64d8"},
    {"user_id", "123456"}
};

absmartly::ContextData data = nlohmann::json::parse(json_response)
    .get<absmartly::ContextData>();

absmartly::Context context(config, data);
assert(context.is_ready());

File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/create-new-context-request/kotlin/sync.kt
val contextConfig = ContextConfig.create()
    .setUnit("session_id", "bf06d8cb5d8137290c4abb64155584fbdb64d8")
    .setUnit("user_id", "123456")

val context = sdk.createContext(contextConfig)
    .waitUntilReady()
